base/shell/explorer/lang/et-EE.rc

```diff
@@ -0,0 +1,208 @@
+/*
+ * PROJECT:     EXPLORER for ReactOS
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * FILE:        base/shell/explorer/lang/et-EE.rc
```

Comment:
Please remove the "FILE" line.

Reply:
Heh, I guess I need to update RAPPS headers and the Wiki article :p